README.md

```diff
@@ -50,3 +50,28 @@ ConfluentSchema.validate(payload, "my-subject")
 ```
 
 Check out `ConfluentSchema.child_spec/1` for documentation about all options.
+
+## Test
```

Comment:
it would be sweet if there was a mix task which could fetch the config from confluent for a particular topic and stash it in `priv/`. Similar to `mix hex.docs offline`

Reply:
Agreed. In fact, I did write code for that, to get the schemas for myself, and then I discarded 😅....it should be simple!